internal/services/fetch.go

```diff
@@ -19,7 +19,7 @@ func fetch(service string, url string, headers map[string]string, extract map[st
 	start := time.Now()
 	req, err := http.NewRequest("GET", url, nil)
 	if err != nil {
-		logrus.Warningf("Error trying to request %s: %s", url, err.Error())
+		logrus.Errorf("Error trying to request %s: %s", url, err.Error())
```

Comment:
`err != nil` here so `req == nil`, we will dereference a `nil` pointer at line 27